dev-python/clade/clade-3.3.7.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Tool for extracting information about software build process and source code"
+HOMEPAGE="https://pypi.org/project/clade/"
```

Comment:
Add GitHub to [metadata](https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/index.html) (<remote type="github-id">).